src/style.css

```
@@ -1,3 +1,63 @@
+:root {
+  --color-primary: #00acdc;
+  --color-black: black;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__image img {
+  display: block;
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```

1. move `position: relative;` to `.is-active`
2. you can use `:not(:first-child)`

```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

---

done